package.json

```diff
@@ -1782,6 +1782,16 @@
                         "daily"
                     ],
                     "scope": "application"
+                },
+                "jupyter.variableTooltipFields": {
```

Comment:
Ran this setting past jmew and greazer, they thought having a setting for this was fine. We may be changing which attributes we show users based on a QP anyway.

Reply:
It is a bit of a complex setting. Not sure how many customers would understand it based on the description. But doesn't hurt to have I suppose. Even if no one external uses it, it keeps us thinking in a more extensible way.

Reply:
Having keyed by language when the description say that it's only for python is a bit funny.

Reply:
Yeah, that's true. I do think it should be keyed by language but I don't believe we support executing interactive cells in other languages in the IW today, so I didn't want the description to imply that other languages should work now.

Reply:
I personally think we shouldn't be adding this. Based on what I've learnt over the years & what other PMs have said, this isn't the best way. To quote a famous man `You don't hand out a cookie to someone & ask if they want a cookie`. 
But, not going to block this PR for this.

Reply:
Would we remove this property if its not used?
If 10 out of 10,000 people use this, would we still keep this property or remote it.

The draw back of adding such properties is the properties window gets overloaded with 1000s of items (as reported by plenty of PMs as well).
@jmew /cc

Reply:
Why not remove this from `package.json` & we support it.
Just as we support other settings but not advertised in `package.json`
Blocking based on the request that we remove this from here & just add support for it.

We do this today for `"jupyter.disableZMQSupport"`, its a property that is supported in settings.json but not advertised, so users don't see it.

@joyceerhl @jmew (this way we can customize it for QPs & never have to add it unless users need it).

Reply:
Good idea, I'm fine with that.